src/nvim/ex_docmd.c

```diff
@@ -9224,6 +9224,24 @@ static void ex_filetype(exarg_T *eap)
     EMSG2(_(e_invarg2), arg);
 }
 
+/// Same as :filetype plugin indent enable
+/// Updates the state used for :filetype without args.
+void force_enable_filetype(void)
+{
+  if (!filetype_detect) {
+    source_runtime((char_u *)FILETYPE_FILE, true);
+    filetype_detect = true;
+  }
+  if (!filetype_plugin) {
+    source_runtime((char_u *)FTPLUGIN_FILE, true);
+    filetype_plugin = true;
+  }
+  if (!filetype_indent) {
+    source_runtime((char_u *)INDENT_FILE, true);
+    filetype_indent = true;
+  }
```

Comment:
@fmoralesc Since this logic is basically already done in `ex_filetype`, why not just `do_cmdline_cmd("filetype plugin indent enable")` ?